applications/StructuralMechanicsApplication/custom_python/add_custom_constitutive_laws_to_python.cpp

```diff
@@ -45,7 +52,7 @@ using namespace pybind11;
 
 void  AddCustomConstitutiveLawsToPython(pybind11::module& m)
 {
-
+	
```

Comment:
please do not introduce whitespace changes

=> I recommend setting your ide to deleting trailing whitespaces on saving